### PR TITLE
template: add stupsNATSubnets function

### PR DESCRIPTION
**Pull request by aermakov-zalando**
Because we shouldn't be setting it manually every time.

### CODE REVIEW

**Comment by aermakov-zalando** [comment]
👍

File: provisioner/template_test.go [hunk 1]
```diff
@@ -464,3 +464,33 @@ DwIDAQAB
...
+			subnets: "172.31.64.0/28,172.31.64.16/28,172.31.64.32/28",
+		},
+		{
+			vpc:     "10.153.192.0/19",
+			subnets: "10.153.200.0/28,10.153.200.16/28,10.153.200.32/28",
+		},
+
+		{
+			vpc:     "10.149.64.0/19",
+			subnets: "10.149.72.0/28,10.149.72.16/28,10.149.72.32/28",
```
Comment (szuecs) [inline provisioner/template_test.go]: Maybe error cases would be also good to check, such that we are sure that the user input won't trigger weird bugs.

Comment (aermakov-zalando) [reply provisioner/template_test.go]: Forgot to add the tests. Done.

**Comment by szuecs** [comment]
:+1:

**Comment by aermakov-zalando** [comment]
👍

**Pull request merged**